hedera-node/hedera-file-service/src/main/java/com/hedera/node/app/service/file/FileMetadata.java

```diff
@@ -16,26 +16,25 @@
 
 package com.hedera.node.app.service.file;
 
+import com.hedera.hapi.node.base.FileID;
 import com.hedera.hapi.node.base.KeyList;
 import com.hedera.hapi.node.base.Timestamp;
 import com.hedera.pbj.runtime.io.buffer.Bytes;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.Nullable;
 
-// TODO : Remove use of TopicMetadata and change to use File instead
-
 /**
  * File metadata
  *
- * @param fileNumber               file's memo
+ * @param fileId                   file's ID
  * @param expirationTimestamp      file's expiration time
  * @param keys                     file's sign keys
  * @param contents                 file bytes that are the contents of the file
  * @param memo                     file's memo
  * @param deleted                  file's deleted flag
  */
 public record FileMetadata(
-        long fileNumber,
+        FileID fileId,
```

Comment:
Do we need `FileMetadata` . Can we not use `File` object alone ? Not a blocker for this PR, just something to think about

Reply:
We can, I think but it is a similar path like in topic. let's create an issue to handle both.